src/project_handler_factory.py

```diff
@@ -1,13 +1,13 @@
 from src.services.inmem_project_handler import InMemProjectHandler
 from dotenv import load_dotenv
 import os
+from src.services.db_project_handler import DbProjectHandler
 
 load_dotenv()
-LOCAL_STORAGE = os.getenv('LOCAL_STORAGE') == 'True'
+LOCAL_STORAGE = 1 if os.getenv("LOCAL_STORAGE") == "True" else 0

```

Comment:
Could be simplified: 
LOCAL_STORAGE = os.getenv("LOCAL_STORAGE") == "True"
or
LOCAL_STORAGE = os.getenv("LOCAL_STORAGE) in ("True", 1, "1", true) (just for checking all possible values)